packages/itwinui-css/src/table/condensed-size.scss

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+
+@mixin iui-table-condensed-size($condensedSize) {
+  @if ($condensedSize == condensed) {
+    --_iui-table-condensed-header-size: #{$iui-baseline * 4};
+    --_iui-table-condensed-row-size: #{$iui-baseline * 4 + 2px};
+  } @else {
+    --_iui-table-condensed-header-size: #{$iui-baseline * 3};
+    --_iui-table-condensed-row-size: #{$iui-baseline * 3 + 2px};
+  }
+
+  .iui-table-header .iui-cell,
+  .iui-paginator {
+    min-height: var(--_iui-table-condensed-header-size);
```

Comment:
These are duplicated in the compiled CSS so it lost the point of using CSS variables.
Maybe just hardcode sizes in this file:
```scss
@mixin iui-table-condensed-size() {
  .iui-table-condensed {
    --_iui-table-condensed-header-size: #{$iui-baseline * 4};
    --_iui-table-condensed-row-size: #{$iui-baseline * 4 + 2px};
  } 

  .iui-table-extra-condensed {
    --_iui-table-condensed-header-size: #{$iui-baseline * 3};
    --_iui-table-condensed-row-size: #{$iui-baseline * 3 + 2px};
  }

  .iui-table-header .iui-cell,
  .iui-paginator {
    min-height: var(--_iui-table-condensed-header-size);
  ...
```

Reply:
I tried using your suggested code but ran into some issues when dealing with tables that weren't condensed. I tweaked the file a bit to remove duplication in the CSS.

Reply:
Is it possible to set default sizes too, so that specificity would be even lower because it is kind of the same as before.

Reply:
Default values have been updated. The test images for condensed tables have changed as a result, but I believe it is for the better.

In our current implementation of condensed tables, the `.iui-table-header .iui-cell` selector was being overriden by the `.iui-row .iui-cell` selector even on cells that were part of the header. Look at the following example of a header cell of an extra condensed table

![Extra condensed screenshot](https://user-images.githubusercontent.com/78564221/178734433-8bc217b9-6964-419d-aa94-72210ae2e0d0.PNG)



Now, our calculated value for the `min-height` for table headers in condensed tables is `2px` less than the `min-height` for the body cells. This is consistent with our non-condensed table

There are other slight visual changes as a result of this push, but I'm not entirely sure what's going on there.